pytorch_lightning/plugins/environments/torchelastic_environment.py

```diff
@@ -55,18 +53,16 @@ def main_port(self) -> int:
             os.environ["MASTER_PORT"] = "12910"
         log.debug(f"MASTER_PORT: {os.environ['MASTER_PORT']}")
 
-        port = int(os.environ.get("MASTER_PORT"))
-        return port
+        return int(os.environ["MASTER_PORT"])
 
     @staticmethod
     def detect() -> bool:
         """Returns ``True`` if the current process was launched using the torchelastic command."""
         required_env_vars = {"RANK", "GROUP_RANK", "LOCAL_RANK", "LOCAL_WORLD_SIZE"}
         return required_env_vars.issubset(os.environ.keys())
 
-    def world_size(self) -> Optional[int]:
-        world_size = os.environ.get("WORLD_SIZE")
-        return int(world_size) if world_size is not None else world_size
```

Comment:
Is that possible that word_size be None?

Reply:
No. WORLD_SIZE should always be made available by torch distributed elastic: https://pytorch.org/docs/stable/elastic/run.html